doc/kb_sdk_impl_methods.md

```diff
@@ -0,0 +1,201 @@
+# <A NAME="top"></A>![alt text](https://avatars2.githubusercontent.com/u/1263946?v=3&s=84 "KBase") [KBase SDK](../README.md)
+
+1. [Install SDK Dependencies](kb_sdk_dependencies.md)
+2. [Install SDK with Docker](kb_sdk_dockerized_install.md)
+3. [Create Module](kb_sdk_create_module.md)
+4. [Edit Module and Method(s)](kb_sdk_edit_module.md)
+5. **Implement Method(s)**
+6. [Specify User Interface](kb_sdk_make_ui.md)
+7. [Locally Test Module and Method(s)](kb_sdk_local_test_module.md)
+8. [Register Module](kb_sdk_register_module.md)
+9. [Test in KBase](kb_sdk_test_in_kbase.md)
+10. [Complete Module Info](kb_sdk_complete_module_info.md)
+11. [Deploy](kb_sdk_deploy.md)
+
+#### <A NAME="impl"></A>5. Edit Impl file
+
+In the lib/\<MyModule\>/ directory, edit the <MyModule>Impl.py (or *.pl) "Implementation" file that defines the methods 
```

Comment:
Based on the examples, it looks like for java the implementation happens in the server file. I'll note that.